metadata-ingestion/src/datahub/cli/specific/dataproduct_cli.py

```diff
@@ -332,6 +336,7 @@ def add_owner(urn: str, owner: str, owner_type: str) -> None:
     if not urn.startswith("urn:li:dataProduct:"):
         urn = f"urn:li:dataProduct:{urn}"
     dataproduct_patcher: DataProductPatchBuilder = DataProduct.get_patch_builder(urn)
+    owner_type, owner_type_urn = validate_ownership_type(owner_type)
```

Comment:
is `owner_type_urn` unused here?

Reply:
Good catch, as I mentioned in description, via this command we can not add custom ownership because we are bound by choices list. Nevertheless the variable should not be left unused, I added it to the `OwnerClass`. Thanks!